### PR TITLE
Remove redundant files. Fixes #8966

### LINKED ISSUE #8966: python setup.py clean should remove the pdfs generated by the docs.
```
	doc/src/modules/physics/vector/kin_1.pdf
	doc/src/modules/physics/vector/kin_1pt.pdf
	doc/src/modules/physics/vector/kin_2.pdf
	doc/src/modules/physics/vector/kin_2pt.pdf
	doc/src/modules/physics/vector/kin_3.pdf
	doc/src/modules/physics/vector/kin_4.pdf
	doc/src/modules/physics/vector/kin_angvel1.pdf
	doc/src/modules/physics/vector/kin_angvel2.pdf
	doc/src/modules/physics/vector/kin_angvel3.pdf
	doc/src/modules/physics/vector/kin_rolling.pdf
	doc/src/modules/physics/vector/simp_rot.pdf
	doc/src/modules/physics/vector/vec_add.pdf
	doc/src/modules/physics/vector/vec_cross.pdf
	doc/src/modules/physics/vector/vec_dot.pdf
	doc/src/modules/physics/vector/vec_fix_notfix.pdf
	doc/src/modules/physics/vector/vec_mul.pdf
	doc/src/modules/physics/vector/vec_rep.pdf
	doc/src/modules/physics/vector/vec_simp_der.pdf
```

It would also be nice if they were included in the `.gitignore` file.

### PR DESCRIPTION
**Pull request by darshanime**
The files generated by the docs are removed. 
Fixes #8966.

### CODE REVIEW

**Comment by darshanime** [comment]
@moorepants, please review.

File: .gitignore [hunk 1]
```diff
@@ -67,3 +67,23 @@ sample.tex
 
 # IPython Notebook Checkpoints
 .ipynb_checkpoints/
+
+#redundant files
+doc/src/modules/physics/vector/kin_1.pdf
```
Comment (moorepants) [inline .gitignore]: Maybe use a glob here: `doc/src/modules/physics/vector/*.pdf`

File: setup.py [hunk 2]
```diff
@@ -190,6 +190,20 @@ def run(self):
             elif os.path.isdir(f):
                 shutil.rmtree(f)
 
+        os.chdir(dir_setup+"/doc/src/modules/physics/vector")
```
Comment (moorepants) [inline setup.py]: Can a glob work to simply get all the pdf files in that directory?

**Comment by darshanime** [comment]
@moorepants done.

File: setup.py [hunk 3]
```diff
@@ -190,6 +191,11 @@ def run(self):
             elif os.path.isdir(f):
                 shutil.rmtree(f)
 
+
+        for name in glob.glob(dir_setup+"/doc/src/modules/physics/vector/*.pdf"):
+            if os.path.isfile(name):
+                os.remove(name)
```
Comment (debugger22) [inline setup.py]: `glob` only works on UNIX based systems. Use `os` module instead of `glob`.

Comment (moorepants) [inline setup.py]: Why do you think glob only works on UNIX based systems? The docs don't say there are any limitations: https://docs.python.org/2/library/glob.html. Where is this limitation discussed?

File: setup.py [hunk 4]
```diff
@@ -190,6 +191,11 @@ def run(self):
             elif os.path.isdir(f):
                 shutil.rmtree(f)
 
+
+        for name in glob.glob(dir_setup+"/doc/src/modules/physics/vector/*.pdf"):
```
Comment (moorepants) [inline setup.py]: It is best to use `os.path.join` to build up path strings in a cross platform way.

Comment (darshanime) [inline setup.py]: True. Made the change !

File: setup.py [hunk 5]
```diff
@@ -190,6 +191,10 @@ def run(self):
             elif os.path.isdir(f):
                 shutil.rmtree(f)
 
+        for name in glob.glob(os.path.join(dir_setup,"doc","src","modules","physics","vector","*.pdf")):
```
Comment (moorepants) [inline setup.py]: please add spaces after commas, see pep 8. And wrap the line if it is longer than 79 characters.

**Comment by moorepants** [comment]
This looks good. No tests are run, so I'm merging.

File: setup.py (hunk 3 above)
Comment (debugger22) [inline setup.py]: I had a problem with `glob` on windows an year ago so I had to use `os.listdir` and `os.walk`. I don't remember the cause and the reference that I used. Sorry for blind suggestion.

**Pull request merged**